### ISSUE TITLE
training time can't use gpu

### ISSUE THREAD

**Issue by 1029694141** [potsawee/longsum0#4]
hi,I followed previous steps and combine data in one file.
when i run
python train/train_abssum.py 
then

#parameters: 409436160
loaded: /home/WwhStuGrp/yyfwwhstu16/longsum/data/arxiv_trainingtimecombine/arxiv_train.pk.bin
loaded: /home/WwhStuGrp/yyfwwhstu16/longsum/data/arxiv_trainingtimecombine/arxiv_val.pk.bin
Shuffle data...
[2021-11-26 18:58:51.705232] step 0/20000000: loss = 7.49640

my cpu CPU usage becomes full and  gpu is not used

note：
1.I had set up the cuda environment, and other experiments can use gpu. 
i.e:
Python 3.7.11 (default, Jul 27 2021, 14:32:16)
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> torch.cuda.is_available()
True

2.I also tried CUDA_VISIBLE_DEVICES=0,1,2,3 python train/train_abssum.py  ,it's not working

Looking forward to your reply, thank you very much！！！

**Comment by potsawee** [comment]
Hi @1029694141 , 

Sorry for this late reply, I just saw your question. If you're still looking for some answers, here would be my suggestions

1. Have you set ```use_gpu = True``` in the config file? 
2. Check if you can ```bart_model.cuda()``` this should simply move the model to your GPU.

Let me know if there is still an issue.

**Comment by 1029694141** [comment]
@potsawee 
Thank you for your reply! 
One more question:
How do I run on multiple GPUs? There is no corresponding setting in the conf file

**Comment by potsawee** [comment]
The current code doesn't support multiple-GPU training. However, if you would like to do **DataParallelism** on multiple GPUs, it's quite straightforward and you can just modify the code as needed, e.g.

```
if torch.cuda.device_count() > 1:
     print("Multiple GPUs: {}".format(torch.cuda.device_count()))
     model = nn.DataParallel(model)
```

Please see please see this PyTorch documentation: https://pytorch.org/docs/stable/generated/torch.nn.DataParallel.html for more information.
